housekeeper/store/api/crud/create.py

```diff
@@ -0,0 +1,94 @@
+"""This module handles adding things to the store"""
+
+import datetime as dt
+import logging
+from typing import Dict, List, Tuple
+
+from housekeeper.store import models
```

Comment:
So that all over this file we can suppress `models.`
```suggestion
from housekeeper.store.models import Bundle, Version, File, Tag
```